client/src/components/cityview/realm/villagers/panels/discussion/NpcChatMessage.tsx

```diff
@@ -86,7 +83,7 @@ export const NpcChatMessage = (props: NpcChatMessageProps) => {
           </div>
         </div>
       </div>
-      {showNpcStats && <NpcPopup onClose={() => setShowNpcStats(false)} npc={npc} />}
+      {showNpcStats && <NpcPopup onClose={() => setShowNpcStats(false)} />}
```

Comment:
This would create a popup for each villager in the discussion, thus making the user able to open a lot of them. What do you think about setting the NpcPopup higher in the div elements hierarchy to replace the content of the popup instead of opening a new one, much like the implementation in the villagers panel ?